src/utils/usageReportConsent.js

```diff
@@ -0,0 +1,9 @@
+const KEY = 'usage-data-consent'
+
+export const getConsentStatus = () => {
+  return (localStorage.getItem(KEY) === 'true')
+}
+
+export const setConsentStatus = (consent) => {
+  localStorage.setItem(KEY, consent)
```

Comment:
Line 4 above expect the stored value to be exactly `true` or `'true'`. However, this function accepts basically anything. In other words, encapsulation is not air tight. One way to enforce a boolean value here is to use `!!consent`. It will always resolve to `true` or `false`.

Reply:
Yea that makes sense, updated `setConsentStatus` accordingly. I'd say we should just be cautious of not introducing the edge case below in the future:

`const myBool = !!"false";  // == true`
https://stackoverflow.com/questions/263965/how-can-i-convert-a-string-to-boolean-in-javascript